### PR TITLE
0.2.0v

### PR DESCRIPTION
**Pull request by SauloSilva**
#### Features

- Adding each serializer
- Adding sinatra, sinatra-contrib and sinatra-activerecord with dependencies

### CODE REVIEW

**Comment by adrianotadao** [comment]
@SauloSilva for sure. At night, I'll take a look :D

File: Gemfile [hunk 1]
```diff
@@ -2,16 +2,12 @@ source 'http://rubygems.org'
...
-  gem 'sinatra', '1.4.5'
-  gem 'sinatra-contrib', '1.4.2'
-  gem 'rake', '10.4.2'
-  gem 'json-schema', '2.5.0'
   gem 'rspec', '3.1.0'
-  gem 'sinatra-activerecord', '2.0.4'
-  gem 'sqlite3', '1.3.10'
-  gem 'database_cleaner', '1.4.0'
+  gem 'json-schema', '2.5.0'
+  gem 'coveralls', require: false
```
Comment (adrianotadao) [inline Gemfile]: @SauloSilva You are missing the gem version...

File: db/migrate/00_create_tests.rb [hunk 2]
```diff
@@ -1,7 +0,0 @@
-class CreateTests < ActiveRecord::Migration
```
Comment (adrianotadao) [inline db/migrate/00_create_tests.rb]: @SauloSilva :clap: :clap:

**Comment by adrianotadao** [comment]
@SauloSilva It's just perfect. :) Go ahead with the merge! ... I just saw a very small detail... you just is forgetting the gem version on Gemfile

File: sinatra-active-model-serializers.gemspec [hunk 3]
```diff
@@ -13,33 +13,43 @@ Gem::Specification.new do |s|
   s.license = 'MIT'
 
   # generated from git shortlog -sn
-  s.authors = ['saulosantiago']
+  s.authors = ['Saulo Santiago', 'Adriano Tadao']
```
Comment (adrianotadao) [inline sinatra-active-model-serializers.gemspec]: @SauloSilva  Ohh man! In this way you'll make me cry! =D This gem is your son! Is a result of your big dedication, but, thanks for your consideration. =D

Comment (SauloSilva) [inline sinatra-active-model-serializers.gemspec]: @adrianotadao all help deserve to be rewarded in any way. You have been contributor and has improved this code, thanks man :heart:

**Comment by SauloSilva** [comment]
@adrianotadao thanks man, 
I finished. You could merge?

**Comment by SauloSilva** [comment]
@adrianotadao thanks man.

**Comment by adrianotadao** [comment]
@SauloSilva is we! :facepunch: :D

**Pull request merged**